### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by hshahi** [naver/egjs-flicking-plugins#44] [General]
Hi I was wondering if you have an angular example that represent dark mode, or how to change the look and feel

**Comment by malangfox** [comment]
Hello @hshahi.

Our component does not support additional dark mode themes.
You may have to use your own style and apply it as a dark theme.

**Reply by hshahi** [reply]
Thank you but just would like to see an example how easy it is to apply different background

**Reply by malangfox** [reply]
Would you like to change the background color of the panel? Or does it mean the background color of the entire flicking area?
If you want to make the panel color change in dark mode, you can define a style that applies to the class of panels inside the flicking.
In case of [this](https://codesandbox.io/s/flicking-toggle-dark-mode-mk1bdg?file=/src/app/app.component.ts) demo, the style of dark mode is defined separately in `panel` class.

This demo manages dark mode in app.component.ts, but I also like to manage this in service.

**Reply by hshahi** [reply]
Thank you, it is more like entire flicking background is black or dark blue and items back ground dark gray at very least, I think that is the only 2 that needs to change to make it proper dark mode, Am I right?

**Reply by malangfox** [reply]
Proper style of dark mode will depend on how you use Flicking component.
You may choose the parts you want to change when in dark mode, I think most of it is the content inside of panel or panel itself.
